style.css

```diff
@@ -338,4 +373,77 @@
     align-self: center;
     margin: 0 42px 0 0;
   }
+
+  .languages {
+    color: #000;
+    font-weight: 600;
+    font-size: 20px;
+    padding-left: 3vw;
+    margin-top: 4vh;
+    margin-bottom: 4vh;
+    line-height: 24px;
+  }
+
+  .contact-form {
+    width: 100%;
+    height: 120vh;
+    background-color: #6070ff;
+  }
+
```

Comment:
- [ ] As shown from the project requirement please make use of flexbox to align the contents of this page. :+1: 
- project requirement 
![fl](https://github.com/Developer-Kingz/my-portforlio-project/assets/89055502/90c7c78a-b8e3-4cde-a464-0d7f83ee260a)